src/Value/Arithmetic/Percentage.php

```diff
@@ -4,13 +4,21 @@
 namespace MyOnlineStore\Common\Domain\Value\Arithmetic;
 
 use Doctrine\ORM\Mapping as ORM;
+use Litipk\BigNumbers\Decimal;
 use MyOnlineStore\Common\Domain\Exception\InvalidArgument;
 
 /**
  * @ORM\Embeddable
  */
 final class Percentage extends Number
 {
+    /**
+     * @ORM\Column(name="percentage", type="decimal", precision="5", scale="2")
+     *
+     * @var Decimal
```

Comment:
This doesn't work due to the `Decimal` object. There's a custom `BignumbersType` type within the monolith that can be used.